### PR TITLE
updates checkout thumbnails to reflect relevant variant images

### PR DESCRIPTION
**Pull request by Satloff**
I want to merge this change because right now at checkout the thumbnail shown beside an line item is the default product image. Instead, it should be the variant image of the item in the cart. 

I also modified the OrderLine model to include the variant of the order item. This allows for better tracking of data, and simpler referencing for future features. 

### Pull Request Checklist

(Please keep this section. It will make maintainer's life easier.)

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] Python code quality checks pass: `pycodestyle`, `pydocstyle`, `pylint`.
1. [ ] JavaScript code quality checks pass: `eslint`.

### CODE REVIEW

**Comment by Satloff** [comment]
@elwoodxblues will you take a look?

File: saleor/order/utils.py [hunk 1]
```diff
@@ -188,7 +189,7 @@ def move_order_line_to_group(line, target_group, quantity):
             delivery_group=target_group, product=line.product,
             product_name=line.product_name, product_sku=line.product_sku,
             quantity=quantity, unit_price_net=line.unit_price_net,
-            stock=line.stock,
+            stock=line.stock, variant=product_variant,
```
Comment (elwoodxblues) [inline saleor/order/utils.py]: What is the `product_variant` variable here? I don't see where it's defined. I think you meant `line.variant`.

File: templates/order/_ordered_items_table.html [hunk 2]
```diff
@@ -51,9 +51,10 @@ <h3>
     <div class="table__row order-details__product{% if forloop.last %} order-details__last-row{% endif %}">
       <div class="row">
         <div class="col-md-6 col-9">
+          <p>{{item_line.product}}</p>
```
Comment (elwoodxblues) [inline templates/order/_ordered_items_table.html]: We already display the product name (line 59), so I think this is not necessary.

File: saleor/static/scss/layouts/_cart.scss [hunk 3]
```diff
@@ -58,6 +58,10 @@
       svg {
         vertical-align: text-top;
       }
+      svg:hover path{
```
Comment (elwoodxblues) [inline saleor/static/scss/layouts/_cart.scss]: This change seems unrelated to the issue being solved in this PR. I'd rather remove it and consider in a separate ticket if we should highlight the SVG icons.

**Comment by elwoodxblues** [comment]
Thanks for the PR @Satloff. Indeed, we should render the variant images in the cart. I've just submitted a review. I think there are few small changes to be made. Also, there is one conflicting file to be resolved before we can merge it.

**Comment by JDeuce** [comment]
Any update on this? I started playing with Saleor and my variant images don't seem to do anything. What else do they do if not update the product and checkout page?

I was also hoping it would update the image on the product page when they selected the attribute for that variant. Like if you have variants for color and you select Red, it should show the red image... and if you select blue, it switches to the blue image for that variant... before you add it to the cart.

File: templates/order/_ordered_items_table.html [hunk 4]
```diff
@@ -51,9 +51,10 @@ <h3>
...
           <a class="link--clean" href="{{ item_line.product.get_absolute_url }}">
-            <img src="{% product_first_image item_line.product size="60x60" method="crop" %}" 
-            srcset="{% product_first_image item_line.product size="60x60" method="crop" %} 1x, {% product_first_image item_line.product size="120x120" method="crop" %} 2x" 
+            <img src="{% get_thumbnail item_line.variant.get_first_image size="60x60" method="crop" %}"
```
Comment (NyanKiyoshi) [inline templates/order/_ordered_items_table.html]: What if the variant was deleted, and thus `item_line.variant` is `None`?

**Comment by maarcingebala** [comment]
@JDeuce This PR needs to be rebased against latest master and there were also a few problems in the code to fix. @Satloff Will you have time to continue working on this in the nearest future? If not, I'll try to have someone from our team to continue this PR but it won't happen sooner than in two weeks, I suppose.

**Comment by Pacu2** [comment]
I'll address this issue in another PR
